### PR TITLE
Adding HTMLAssignment3.html

### PR DESCRIPTION
**Pull request by srigowrideo**

### CODE REVIEW

File: Srigowri_Deogirikar/Contactuspage.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+    <!Doctype html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE-edge">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Assignment 3</title>
+        
+    </head>
+    <style>
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/Contactuspage.html:10]: style tag needs to be in head tag

File: Srigowri_Deogirikar/HTMLAssignment3.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <meta http-equiv="X-UA-Compatible" content="IE-edge">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0">
+      <title>Assignment 3</title>
+      
+  </head>
+  <body>
+      <h1>Ambersoft</h1>
+      <ol>
+  <li style="color:rgb(77, 44, 44)">MI-MEAN</li>
+
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/HTMLAssignment3.html:14]: ol tag closing is missing

File: Srigowri_Deogirikar/HTMLAssignment3.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      
+  </head>
+  <body>
+      <h1>Ambersoft</h1>
+      <ol>
+  <li style="color:rgb(77, 44, 44)">MI-MEAN</li>
+    
+    <ol style="list-style-type:upper-roman;">
+      <li style="color:rgb(77, 44, 44)"> MI-MEAN-6</li>
+
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/HTMLAssignment3.html:17]: ol tag closing is missing

File: Srigowri_Deogirikar/HTMLAssignment3.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      <li style="color:rgb(62, 55, 146)">Anu</li>
+      <li style="color:rgb(62, 55, 146)">Dipti</li>
+  </ul>
+  <li style="color:rgb(62, 55, 146)">SYSTEM-2</li>
+    <ul>
+      <li style="color:rgb(62, 55, 146)">Triveni</li>  
+      <li style="color:rgb(62, 55, 146)">Kalyani</li>
+  </ul>  </ol> </ol></ol>
+  </body>
+  </html>
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/HTMLAssignment3.html:54]: create proper indentation for all html files

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
create proper indentation for all html files

File: Srigowri_Deogirikar/Contactuspage.html (hunk 1 above)
Comment (srigowrideo) [reply Srigowri_Deogirikar/Contactuspage.html:10]: ok sir

File: Srigowri_Deogirikar/HTMLAssignment3.html (hunk 2 above)
Comment (srigowrideo) [reply Srigowri_Deogirikar/HTMLAssignment3.html:14]: sir its at the last.

Comment (srigowrideo) [reply Srigowri_Deogirikar/HTMLAssignment3.html:14]: then only i will get the proper numbering

File: Srigowri_Deogirikar/HTMLAssignment3.html (hunk 3 above)
Comment (srigowrideo) [reply Srigowri_Deogirikar/HTMLAssignment3.html:17]: sir its at the last

File: Srigowri_Deogirikar/HTMLAssignment3.html (hunk 4 above)
Comment (srigowrideo) [reply Srigowri_Deogirikar/HTMLAssignment3.html:54]: ok sir

**Review by srigowrideo — COMMENTED** [review]
sir, i changed

**Pull request merged**